src/Marketplace/MarketplaceScreen.js

```diff
@@ -92,69 +96,173 @@ export default function MarketplaceScreen({ navigation }) {
         produce.fields.Favorited = false;
```

Comment:
There might be an issue with the Favorited toggle, I'm getting an unknown field error when I toggle. 
<img width="182" alt="favourite" src="https://user-images.githubusercontent.com/28603722/163326318-cb4aa6c7-23a7-422d-a4ae-23596c5087f0.PNG">
<img width="193" alt="unknown field name favourite" src="https://user-images.githubusercontent.com/28603722/163326349-872d5ebd-7f4f-4058-b8ed-a5d8bda58851.PNG">